script/Ciel/Day 3/03_01_CIEL03_2_1.txt

```diff
@@ -401,29 +404,28 @@ As I watch the two of them out of the corner of my eye, I gaze idly at the scene
 [sha:a7a08f0aef4874a611ab2c226021bdb030e403df]{
 -- Page 44, Offset 4960. Mods: @k, @e.
 -- 「そんなことはないッス。先輩は他のおんなどもに比べて発育いいんスから。
-// 摂理 is the actual word but I guess fuel efficient makes more sense
 "That's not true. You're a lot more developed than the other girls.
 }
 [sha:5640c1d8727735d842bd3d7891e9dd0a7867b42d]{
 -- Page 44, Offset 4962. Glued. Mods: @x.
 -- ……が大きいぶん、燃費が悪いのは自然の摂理っスよ」
-// 摂理 is the actual word but I guess fuel efficient makes more sense
+// I have no idea how to work in 摂理
```

Comment:
It's God's will that the bigger you are, the less fuel-efficient you are, ya know? (kidding)

no idea either, the current one seems fine imo